learning.py

```diff
@@ -298,6 +298,18 @@ def sample(self):
             self.sampler = weighted_sampler(list(self.dictionary.keys()),
                                             list(self.dictionary.values()))
         return self.sampler()
+        
+    def remove_sparse(self,sparse_value):
+    	""" keep items with probablity sparse value 
+    	and remove all others , eg if sparse_value=0.98 items with probablity
+    	0.02 would be removed"""
```

Comment:
You are not only removing items with probability 0.02, you are removing items with probability less than 0.02. Also, add a period at the end of the comment.